doc/source/amazon/step-zero-aws.rst

```diff
@@ -57,21 +57,25 @@ The Procedure
 
 #. Set the region to deploy in
 
-      export REGION=`curl -s http://169.254.169.254/latest/dynamic/instance-identity/document|grep region|awk -F\" '{print $4}'`
+    export REGION=`curl -s http://169.254.169.254/latest/dynamic/instance-identity/document|grep region|awk -F\" '{print $4}'`
+
+#. Install the AWS CLI:
+
+    sudo apt-get update
+    sudo apt-get install awscli
 
 #. Set the availability zones for the nodes
 
    For this guide we will be allowing nodes to be deployed in all AZs::
 
        export ZONES=$(aws ec2 describe-availability-zones --region $REGION | grep ZoneName | awk '{print $2}' | tr -d '"')
-       export ZONES=$(echo $ZONES | tr -d " " | rev | cut -c 2- | rev)
```

Comment:
Did you mean to delete this line?

Reply:
I think yes from looking at the screenshot you posted.

Reply:
Yes. Before this line was removed: 

```
ubuntu@ip-172-31-94-142:~$ echo $ZONES
us-east-1aus-east-1bus-east-1cus-east-1dus-east-1eus-east-1
```

Without this line: 

```
ubuntu@ip-172-31-94-142:~$ echo $ZONES
us-east-1a us-east-1b us-east-1c us-east-1d us-east-1e us-east-1f
```